controller/patterns/pattern_config.py

```diff
@@ -33,4 +31,6 @@
     
     # # Effect patterns
     ('0x2', SparklePattern, dict(color=np.array([255, 255, 255]), sparkle_probability=0.001, decay_param=0.95)),
-]
+    ('1x2', RainbowWavesPattern, dict()),
+    ('1x3', BouncingBlocksPattern, dict())
```

Comment:
The button assignment should probably by '2x2'. 

Btw the button layout for launchpad is here:
https://github.com/obeezzy/lpminimk3/blob/2041f0ea04594eaf8461e9f884944556b2d9d5ee/lpminimk3/_core/components.py#L784

Reply:
Updated